elife/hub.sls

```diff
@@ -20,4 +20,15 @@ hub:
         - require:
             - deploy-user
             - cmd: hub
+            - user: {{ pillar.elife.hub.username }}
+
+hub-link-config:
+    cmd.run:
+        - name: |
+            cd $(eval echo "~$username")/
```

Comment:
what is setting `$username` here?

Reply:
ah, now I understand how this (doesn't) work, will rewrite